dags/pull_wys.py

```diff
@@ -83,7 +84,7 @@ def custom_fail_slack_alert(context: dict) -> str:
 # Run at 3 PM local time every day
 
 with wys_postgres.get_conn() as con:
```

Comment:
this is a bit of an unideal pattern that we should've gotten rid of before because this connection gets opened every time the schedule refreshes this file. 

might as well move fully towards the TaskFlow API paradigm and only open the connection within the decorated functions

Reply:
Let's just test opening the connection within the decorated function for the get_schedules task.